terraform_bod/docker_ec2.tf

```diff
@@ -0,0 +1,45 @@
+# The bastion AMI
```

Comment:
Comment should be "The docker AMI".  Probably needs to be updated in the main terraform dir too.